### DISCUSSION TITLE
Get an error when trying to run the "scrape_plain_text_openai" example

### DISCUSSION THREAD

**Discussion by 22sauravt** [ScrapeGraphAI/Scrapegraph-ai#745] [Q&A]
scrapegraphai\nodes\fetch_node.py", line 233, in handle_local_source    return self.update_state(state, compressed_document)
           ^^^^^^^^^^^^^^^^^
AttributeError: 'FetchNode' object has no attribute 'update_state'

**Comment by VinciGit00** [comment]
Use document scraper instead

**Comment by VinciGit00** [comment]
Use document scraper instead
